server/public/model/outgoing_oauth_connection.go

```diff
@@ -110,8 +138,8 @@ func (oa *OutgoingOAuthConnection) IsValid() *AppError {
 	return nil
 }
 
-// IsValidGrantType validates the grant type and its parameters returning an error if it isn't properly configured
-func (oa *OutgoingOAuthConnection) IsValidGrantType() *AppError {
+// HaveValidGrantType validates the grant type and its parameters returning an error if it isn't properly configured
```

Comment:
nit: `HasValidGrantType` would read slightly better since it's a single connection object.